python/pyspark/mllib/feature.py

```diff
@@ -110,6 +110,15 @@ class JavaVectorTransformer(JavaModelWrapper, VectorTransformer):
     """
 
     def transform(self, vector):
+        """
```

Comment:
Have you generated the docs?  I'm wondering if this doc is inherited.  I'd think you would need to use inherit_doc.  (See e.g. NaiveBayesModel)